src/dbSta/src/dbNetwork.cc

```diff
@@ -1670,6 +1676,39 @@ Net* dbNetwork::net(const Term* term) const
   return nullptr;
 }
 
+dbNet* dbNetwork::flatNet(const Term* term) const
+{
+  dbITerm* iterm = nullptr;
+  dbBTerm* bterm = nullptr;
+  dbModITerm* moditerm = nullptr;
+  dbModBTerm* modbterm = nullptr;
+  staToDb(term, iterm, bterm, moditerm, modbterm);
+  if (bterm) {
+    dbNet* dnet = bterm->getNet();
+    if (dnet) {
+      return dnet;
+    }
+  }
+  return nullptr;
+}
+
+dbModNet* dbNetwork::hierNet(const Term* term) const
+{
+  dbBTerm* bterm = nullptr;
+  dbModBTerm* modbterm = nullptr;
```

Comment:
These are both null so this method doesn't do anything but return nullptr.  It doesn't seem right...

Reply:
Thanks, good catch. That is definitely wrong (missing call to populate those things). I have removed the calls to get the flat/hier net from a term which we no longer use.